### DISCUSSION TITLE
Option to reset Users Activated Devices

### DISCUSSION THREAD

**Discussion by surtic86** [Glench/ExtPay#73] [Ideas]
Hello there,

There should be an option to reset the activated devices of a Users.

Problem / Reproduction:
- User creates a new Browser Profile and installs and login into Extension
- User removes the Browser Profile 
- Repeat it for 10 Times

Now the User can't Login into the Extension again since he has already used all his activations.

Im getting more and more of the Report that Users can't Login anymore beside they have a payed Subscription.  They can't even Login to cancel there Subscription.

Thanks
surtic86

**Comment by luizen** [comment]
Any news to this question? It's been 2 years :(

**Reply by Glench** [reply]
Yep, it's been deployed!

**Reply by luizen** [reply]
Great! Thank you!

**Comment by Glench** [comment]
This has been deployed. See the "reset logins for user" button on your ExtensionPay dashboard.
